dispatch.go

```diff
@@ -166,11 +169,11 @@ func (d *dispatch) Call(ctx context.Context, name string, arg interface{}, res i
 }
 
 func (d *dispatch) Notify(ctx context.Context, name string, arg interface{}) error {
-	err := d.dispatchMessage(ctx, MethodNotify, name, arg)
-	if err != nil {
-		return err
-	}
-	d.log.ClientNotify(name, arg)
+	errCh := d.dispatchMessage(ctx, MethodNotify, name, arg)
+	go func() {
+		err := <-errCh
+		d.log.ClientNotify(name, err, arg)
```

Comment:
Hmm weird...it looks fine here, but in the side-by-side view, the d.log line is not lined up with the previous line, but looks like a tab and two spaces. But that might just be a github bug.